gef.py

```diff
@@ -11133,27 +11149,32 @@ def __setup_qemu(self) -> bool:
         return True
 
     def __setup_remote(self) -> bool:
+        success = True
+
         # get the file
-        fpath = f"/proc/{self.pid}/exe"
-        if not self.sync(fpath, str(self.file)):
-            err(f"'{fpath}' could not be fetched on the remote system.")
-            return False
+        if not self.sync(str(self.file)):
+            warn(f"'{fpath}' could not be fetched on the remote system.")
+            success = False
```

Comment:
I think here is a good reason to throw: not having the executable will be very problematic (as explained below) and hard to recover from. 
```suggestion
        if not self.sync(str(self.file)):
            raise FileNotFoundError
```

Reply:
> https://github.com/hugsy/gef/pull/1045#discussion_r1472060932